ttnn/cpp/ttnn/operations/ccl/reduce_scatter/device/reduce_scatter_op.cpp

```diff
@@ -1,11 +1,14 @@
+
+
+
 // SPDX-FileCopyrightText: © 2023 Tenstorrent Inc.
 //
 // SPDX-License-Identifier: Apache-2.0
 
 #include "ttnn/operations/ccl/reduce_scatter/device/reduce_scatter_op.hpp"
 
 #include "ttnn/deprecated/tt_dnn/op_library/reduce/reduce_op.hpp"
-#include "ttnn/operations/ccl/ccl_host_datastructures.hpp"
+#include "ttnn/cpp/ttnn/operations/ccl/ccl_host_datastructures.hpp"
```

Comment:
can we move all such files to ccl/common in some future PR?

Reply:
yes. I'm planning on moving gtests on Friday so I'll move this at the same time. I also have some reorg to do with some of these headers as well (they need to be split up as right now they've become sort of accumulation points for separate classes components).

Reply:
resolved as its a separate work item that I'll start shortly